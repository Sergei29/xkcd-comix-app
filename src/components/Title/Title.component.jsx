import React from "react";
import { TitleContainer, Heading } from "./Title.styles";

const Logo = (
	<a href="https://xkcd.com">
		<img
			src="https://xkcd.com/s/0b7742.png"
			alt="xkcd.com logo"
			height="83"
			width="185"
		></img>
	</a>
);

const Title = () => (
	<TitleContainer>
		{Logo}
		<Heading>A webcomic of romance, sarcasm, math, and language.</Heading>
	</TitleContainer>
);

export default Title;
