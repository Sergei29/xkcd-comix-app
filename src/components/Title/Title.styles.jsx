import styled from "styled-components";

export const TitleContainer = styled.div`
	background-color: #fff;
	border-radius: 10px;
	flex-grow: 2;
	display: flex;
	flex-direction: column;
	margin-left: 2px;
	border: 1.5px solid #071419;
`;
