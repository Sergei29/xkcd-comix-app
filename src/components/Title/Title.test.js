import { shallow } from "enzyme";
import React from "react";

import Title from "./Title.component";

describe("Title", () => {
	let wrapper;

	beforeEach(() => {
		wrapper = shallow(<Title />);
	});

	it("should render the component without crashing", () => {
		expect(wrapper.length).toEqual(1);
	});

	it("should match the snapshot", () => {
		expect(wrapper).toMatchSnapshot();
	});
});
