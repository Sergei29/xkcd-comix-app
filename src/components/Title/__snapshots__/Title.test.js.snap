// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title should match the snapshot 1`] = `ShallowWrapper {}`;
